### PR TITLE
use label

### PR DESCRIPTION
**Pull request by dmaltzan**
https://app.asana.com/0/107495983514632/540494290633443

### CODE REVIEW

File: lib/gtfs_realtime_viz.ex [hunk 1]
```diff
@@ -224,8 +224,8 @@ defmodule GTFSRealtimeViz do
 
   @spec trainify_diff([Proto.vehicle_position], [Proto.vehicle_position], Proto.vehicle_position_statuses, String.t, String.t) :: Phoenix.HTML.Safe.t
   defp trainify_diff(vehicles_base, vehicles_diff, status, ascii_train_base, ascii_train_diff) do
-    base = vehicles_with_status(vehicles_base, status) |> Enum.map(& &1.vehicle && &1.vehicle.id)
-    diff = vehicles_with_status(vehicles_diff, status) |> Enum.map(& &1.vehicle && &1.vehicle.id)
+    base = vehicles_with_status(vehicles_base, status) |> Enum.map(& &1.vehicle && &1.vehicle.label)
```
Comment (dmaltzan) [inline lib/gtfs_realtime_viz.ex]: the thing i'm not quite sure of is this won't really handle if a train doesn't have a label, or if two trains have the same label

**Comment by sghiaa** [comment]
theres a warning about ```warning: variable "formatted_predictions" is unused
  lib/gtfs_realtime_viz.ex:191``` thats been bothering me - can we driveby fix that in this pr?

File: lib/gtfs_realtime_viz.ex (hunk 1 above)
Comment (sghiaa) [reply lib/gtfs_realtime_viz.ex]: why would two trains ahve the same label

Comment (dmaltzan) [reply lib/gtfs_realtime_viz.ex]: they might both be blank

Comment (sghiaa) [reply lib/gtfs_realtime_viz.ex]: maybe we could have a function that takes a vehicle and returns the label or the id if the label is blank

**Pull request merged**